android/filament-android/src/main/java/com/google/android/filament/Stream.java

```diff
@@ -33,13 +33,33 @@
     private long mNativeObject;
     private long mNativeEngine;
 
+    /**
+     * Represents the immutable stream type.
+     */
+    public enum StreamType {
+        /** Not synchronized but copy-free. Good for video. */
+        NATIVE,
+
+        /** Synchronized, but GL-only and incurs copies. Good for AR on older devices. */
```

Comment:
We should be more specific than "older" or "newer", you should mention specific API levels for Android.

Reply:
Done